packages/functions/src/api/get.js

```diff
@@ -14,13 +16,18 @@ const repo = 'https://github.com/web3-storage/w3filecoin'
  * AWS HTTP Gateway handler for GET /version
  */
 export async function versionGet () {
-  const { name , version, commit, stage } = getLambdaEnv()
+  const { PRIVATE_KEY: privateKey } = Config
+  const { did, name, version, commit, stage } = getLambdaEnv()
+  const serviceSigner = getServiceSigner({ did, privateKey })
+  const serviceDid = serviceSigner.did()
+  const publicKey = serviceSigner.toDIDKey()
+
   return {
     statusCode: 200,
     headers: {
       'Content-Type': `application/json`
     },
-    body: JSON.stringify({ name, version, repo, commit, env: stage })
+    body: JSON.stringify({ name, version, did: serviceDid, publicKey, repo, commit, env: stage })
```

Comment:
make it more complete, by including same properties as other APIs like https://up.web3.storage/version